.gitignore

```diff
@@ -88,7 +88,6 @@
 /packages/*/*.log
 /packages/apt/.vagrant/
 /packages/apt/build.sh
-/packages/apt/env.sh
```

Comment:
We don't need to remove this entry because it's still auto generated by https://github.com/apache/arrow/blob/bae2908b3142f1141941d740d42c5d403c01749f/dev/tasks/linux-packages/package-task.rb#L337-L343 .

Reply:
Reverted .gitignore changing.
Thanks